WordPress/Classes/Services/BlogService.m

```diff
@@ -108,12 +124,15 @@ - (Blog *)lastUsedBlog
         return nil;
     }
 
-    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Blog"];
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"visible = YES AND url = %@", url];
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:BlogEntity];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%@ AND url = %@", @"visible = YES", url];
```

Comment:
Great catch, that was me reverting a change with find+replace and it was an artifact. I'll revert it back to being in-line.